packages/tldraw/src/lib/shapes/embed/EmbedShapeUtil.tsx

```diff
@@ -41,7 +40,7 @@ export class EmbedShapeUtil extends BaseBoxShapeUtil<TLEmbedShape> {
 		return true
 	}
 	override canResize(shape: TLEmbedShape) {
-		return !!getEmbedInfo(shape.props.url)?.definition?.doesResize
+		return !!useEmbedDefinition(shape.props.url)?.definition?.doesResize
```

Comment:
```suggestion
		return !!useEmbedDefinition(shape.props.url)?.definition?.doesResize
```

Hmmm this is calling a hook from a non-react context, which could throw if we decide to use `canResize` conditionally, or outside of a react render.

I think probably you need to figure out a way to put all the embed definition info into the embed shape util before the editor mounts so you can access it directly rather than through react context